applications/ShapeOptimizationApplication/python_scripts/mesh_controller_with_solver.py

```diff
@@ -100,4 +100,40 @@ def UpdateMeshAccordingInputVariable(self, variable):
     def Finalize(self):
         self._mesh_moving_analysis.Finalize()
 
+    # --------------------------------------------------------------------------
```

Comment:
this was somehow lost when #2417 was merged...

Reply:
Ok. Hope there are not more lost things popping up.

Reply:
I compared the branches, this should be everything